src/main/java/core/basesyntax/model/CreateStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+import java.time.LocalDate;
+
+public class CreateStrategyImpl implements CreateStrategy {
+    @Override
+    public Fruit createFruit(String fruitType, LocalDate expirationDate) {
```

Comment:
I think it's not the best practice to create such generation. May me it's better to use General class Fruit with field "type"?